VideoLocker/src/main/java/org/edx/mobile/util/Config.java

```diff
@@ -385,6 +389,25 @@ public String getOAuthClientId() {
         return getString(OAUTH_CLIENT_ID);
     }
 
+    /**
+     * @return A list of URIs for updating the app, or an empty list if none are available.
```

Comment:
Suggestion: A javadoc here would be helpful explaining the format of the Uri string/s.